climpred/classes.py

```diff
@@ -153,6 +154,34 @@ def __repr__(self):
         else:
             return _display_metadata(self)
 
+    def plot(self, variable=None, ax=None, show_members=False, cmap=None):
+        """Plot datasets from HindcastEnsemble.
+
+        Args:
+            he (PredictionEnsemble): PredictionEnsemble.
+            variable (str or None): `variable` to show. Defaults to first in data_vars.
+            ax (plt.axes): axis to plot to. Defaults to False. Creates new.
+            show_members (bool): whether to display all members individually.
+                Defaults to True.
+            cmap (str): Name of plt colorbar. Defaults to default.
```

Comment:
I would say that this has to be a `matplotlib` recognized colormap string. Users can't put in `cmocean` or other custom colormaps since we're leveraging the nsamples part of the linear segmented colormap.

Reply:
btw: `cmo.haline` works